support/docs/configuration.md

```diff
@@ -12,17 +12,32 @@ fh_mailer:
             text_template: 'email/consumer_welcome.txt.twig'
             subject: 'Tilburg, je bent er.'
             participants:
-                sender: [ address: 'freshheads@example.com', name: 'Freshheads' ]
+                sender: { address: 'freshheads@example.com', name: 'Freshheads' }
                 from:
-                    - [ address: 'kevin@example.com', name: 'Kevin' ]
+                    - { address: 'kevin@example.com', name: 'Kevin' }
                 reply_to:
-                    - [ address: 'freshheads@example.com', name: 'Freshheads' ]
+                    - { address: 'freshheads@example.com', name: 'Freshheads' }
                 to:
-                    - [ address: 'misha@example.com', name: 'Misha' ]
+                    - { address: 'misha@example.com', name: 'Misha' }
```

Comment:
Ook wel duidelijk maken dat je hier ook een array aan mee kunt geven (of moet?)

Reply:
Het is een array van objecten. Dus de notatie hoe die was werkt zo ie zo niet.
Wat zou jij nog toe willen voegen?

Reply:
ah wacht die is goed zo idd ;)